context/app/static/js/components/Search/SearchWrapper.jsx

```diff
@@ -68,24 +73,8 @@ function makeTableComponent(resultFields, detailsUrlPrefix, idField) {
   };
 }
 
-
-export default function (props) {
-  const {
-    apiUrl, prefixQueryFields, filters, detailsUrlPrefix,
-    idField, resultFields, hitsPerPage, debug, httpHeaders,
-    sortOptions = [{
-      label: 'Relevance',
-      field: '_score',
-      order: 'desc',
-      defaultOption: true,
-    }],
-    hiddenFilterIds = [],
-    searchUrlPath = '_search',
-  } = props;
-  const resultFieldIds = (resultFields.map((field) => field.id)).concat(idField);
-  const searchkit = new SearchkitManager(apiUrl, { httpHeaders, searchUrlPath });
-
-  function MaskedSelectedFilters() {
+function makeMaskedSelectedFiltersComponent(hiddenFilterIds) {
+  return () => {
```

Comment:
Is there a cleaner way to do this rather than returning a nameless function?  It's tough to wrap your head around.

Reply:
Maybe just leave a comment.  Or name the wrapping function so it's clear that it's a component (unless this a paradigm you've seen before!  Then maybe just a note :) )

Reply:
Can we make this a component?

Reply:
I agree, it is confusing, but not sure about better approach. Will comment, and I've filed #564.

(This function-function was there before, it was just buried inside _another_ function... so I don't think this PR is making it any worse.)

Reply:
(Never mind... it's easy. Sorry for the noise.)